graphql.js

```diff
@@ -38,8 +42,8 @@ const resolvers = {
 };
 
 const schema = makeExecutableSchema({
-    typeDefs: [ scalars, Query, Mutation, userSchema, eventSchema ],
-    resolvers: merge(resolvers, userResolvers, eventResolvers),
+  typeDefs: [ scalars, Query, Mutation, userSchema, eventSchema, hospitalSchema, ambulanceSchema],
```

Comment:
(SUPER nitpicky i know but) can we add the space back after `ambulanceSchema` :)

Reply:
Done!